### PR TITLE
Fixes #28255 - upgrade vendor to v3

### PR DESCRIPTION
**Pull request by sharvit**
Main upgrades are `redux-hooks` and `babel` v7

### CODE REVIEW

**Comment by adamruzicka** [comment]
[test foreman_remote_execution]

**Comment by sharvit** [comment]
Updated to use `@theforeman/babel`

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@ locale/*/*.edit.po
 locale/*/*.po.time_stamp
 Gemfile.lock
 node_modules/
+package-lock.json
```
Comment (ekohl) [inline .gitignore]: Could you make sure there's a newline char at the end?

**Comment by sharvit** [comment]
thanks @ekohl, rebased and updated the .gitignore

**Comment by ares** [comment]
Thanks @sharvit and @adamruzicka, merging and will follow with a 3.0.0 release!

**Pull request merged**